muzzle/src/main/java/io/opentelemetry/javaagent/tooling/muzzle/ReferenceMatcher.java

```diff
@@ -304,14 +304,14 @@ private static void collectMethodsFromTypeHierarchy(
   }
 
   // optimization to avoid ArrayList allocation in the common case when there are no mismatches
-  private static List<Mismatch> lazyAdd(List<Mismatch> mismatches, Mismatch mismatch) {
+  static List<Mismatch> lazyAdd(List<Mismatch> mismatches, Mismatch mismatch) {
```

Comment:
`add` for these methods that are not private any more is better name since lazy is an implementation detail.